redhat/README.md

```diff
@@ -0,0 +1,93 @@
+# ODF: kubernetes-csi-addons
+
+Repository where [csi-addons/kubernetes-csi-addons][upstream-k8s-csi-addons] is
+maintained for the OpenShift Data Foundation product.
+
+## Git Repository
+
+### Branches
+
+This GitHub repository contains branches for different product versions.
+
+## Backports
+
+All changes in this repository are *backports* from the [upstream
+project][https://github.com/ceph/ceph-csi]. There should be no functional
```

Comment:
we should not have cephcsi link here

Reply:
Well, there is no point in requesting changes here. These are just backport PRs.
We can suggest and add commits to these or update the main branch and backport from there to avoid these type of issues.

Reply:
🤦🏻

Reply:
Addressed in #87